### PR TITLE
IS-14075: Add mapped datatypes that exist in PowerOffice

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: templates/contactperson.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "_id": "{{@ system @}}-contactperson-collect",
```
Comment (BaardBouvet) [inline templates/contactperson.json]: Should use {{ datatype }}

File: templates/outgoinginvoice.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "_id": "{{@ system @}}-outgoinginvoice-collect",
```
Comment (BaardBouvet) [inline templates/outgoinginvoice.json]: Same here

File: templates/outgoinginvoiceline.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            ]
+          ]
+        ]
+      },
+      "type": "dtl"
+    },
+    "type": "pipe"
+  },
+  {
+    "_id": "{{@ system @}}-outgoinginvoiceline-collect",
```
Comment (BaardBouvet) [inline templates/outgoinginvoiceline.json:46]: where is the rest operation for this datatype?

File: templates/outgoinginvoice.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      "payload_property": "data",
+      "properties": {
+        "datatype": "outgoinginvoice"
+      },
+      "replace_entity": true,
+      "system": "{{@ system @}}",
+      "type": "rest"
+    },
+    {
+      "rules": {
```
Comment (BaardBouvet) [inline templates/outgoinginvoice.json]: Remove id_expression and payload_property if they don't work as expected

File: templates/outgoinginvoiceline.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
+[
```
Comment (BaardBouvet) [inline templates/outgoinginvoiceline.json]: Remove this datatype

**Review by BaardBouvet — CHANGES_REQUESTED** [review]

**Review by BaardBouvet — APPROVED** [review]

File: templates/customer.json [hunk 6]
```diff
@@ -1,18 +1,61 @@
...
+        "ordering": "Name asc",
+        "skip": "0",
+        "top": "20"
+      },
+      "system": "{{@ system @}}",
+      "type": "rest"
+    },
+    "transform": {
+      "properties": {
+        "primary_key": "id",
```
Comment (grove) [inline templates/customer.json:53]: Could you use {{ id }} for the primary key as well?

**Pull request merged**